docs/sources/get-started/index.md

```diff
@@ -0,0 +1,94 @@
+---
+description: Learn how to get started with Explore Traces
+canonical: https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/get-started/
+keywords:
+  - Explore Traces
+  - Get started
+title: Get started with Explore Traces
+menuTitle: Get started
+weight: 300
+---
+
+# Get started with Explore Traces
+
+{{< docs/public-preview product="Explore Traces" >}}
+
+You can use traces to identify errors in your apps and services, and then optimize and streamline them.
+
+When working with traces, start with the big picture. Then drills down using primary signals, RED metrics, filters, and structural or trace list tabs to explore your data. To learn more, refer to [Concepts](../concepts/).
```

Comment:
```suggestion
When working with traces, start with the big picture. Then drill down using primary signals, RED metrics, filters, and structural or trace list tabs to explore your data. To learn more, refer to [Concepts](../concepts/).
```

Reply:
@heitortsergent You and Taylor both had the same suggestion. I merged hers. :)